src/Backend/Core/Js/backend.js

```diff
@@ -672,74 +672,32 @@ jsBackend.controls =
 	// bind confirm message
 	bindConfirm: function()
 	{
-		// variables
-		$askConfirmation = $('.askConfirmation');
-
-		// initialize
-		$askConfirmation.each(function()
+		$('.jsConfirmationTrigger').on('click', function(e)
 		{
-			// get id
-			$this = $(this);
-			var id = $this.data('messageId');
-			var url = $this.attr('href');
-			if(typeof url == 'undefined') url = $this.find('a').attr('href');
+			// prevent default
+			e.preventDefault();
 
-			if(id !== '' && url !== '')
-			{
-				// initialize
-				$('#'+ id).dialog(
-				{
-					autoOpen: false,
-					draggable: false,
-					resizable: false,
-					modal: true,
-					buttons:
-					[
-						{
-							text: utils.string.ucfirst(jsBackend.locale.lbl('OK')),
-							click: function()
-							{
-								// unbind the beforeunload event
-								$(window).off('beforeunload');
+			// get data
+			var href = $(this).attr('href');
+			var message = $(this).data('message');
 
-								// goto link
-								window.location = url;
-							}
-						},
-						{
-							text: utils.string.ucfirst(jsBackend.locale.lbl('Cancel')),
-							click: function()
-							{
-								$(this).dialog('close');
-							}
-						}
-					],
-					open: function(e)
-					{
-						// set focus on first button
-						if($(this).next().find('button').length > 0) $(this).next().find('button')[0].focus();
-					}
-				});
-			}
-		});
+			console.log(message);
```

Comment:
console.log's should be removed, since older versions of Internet Explorer crash on it.